rm/rm-server/src/test/java/functionaltests/nodesource/TestSSHInfrastructureV2.java

```diff
@@ -122,7 +122,9 @@ public static void startSSHServer() throws Exception {
 
         sshd = SshServer.setUpDefaultServer();
 
-        sshd.setKeyPairProvider(new SimpleGeneratorHostKeyProvider());
+        SimpleGeneratorHostKeyProvider keyProvider = new SimpleGeneratorHostKeyProvider();
+        keyProvider.setAlgorithm("RSA");
```

Comment:
RSA-??? . I'm just curious to know whether or not you can specify the number of bits.

Reply:
yep it's possible !